TSKit/Public/TSClient.m

```diff
@@ -149,6 +149,47 @@ - (void)disconnect
     self.isSubscribedToChannels = NO;
 }
 
+- (void)wisperConnect:(nullable NSArray<TSUser *>*)users channels:(nullable NSArray<TSChannel *>*)channels {
+    UInt64 *channelIDs =calloc(users.count + 1, sizeof(UInt64));
+    for (int i = 0; i < users.count; i++) {
```

Comment:
you should be iterating over the `channels` and not `users`

Reply:
Ahh nice catch.